test/e2e/setup-kubeflex.sh

```diff
@@ -16,6 +16,17 @@
 set -x # echo so that users can understand what is happening
 set -e # exit on error
 
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+
+:
+: -------------------------------------------------------------------------
+: Create the hosting kind cluster with ingress controller
+:
+
+kind create cluster --name kubeflex --config ${SCRIPT_DIR}/kind-config.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+kubectl -n ingress-nginx patch deployment/ingress-nginx-controller --patch-file=${SCRIPT_DIR}/nginx-patch.yaml
```

Comment:
This looks dangerous to me. Whenever the aguments in ingress-nginx/main change, the patch has to change too.

Reply:
good point - pinning that to latest tag and ensuring it works with the patch